src/plugins/intel_gpu/src/graph/primitive_inst.cpp

```diff
@@ -1106,6 +1106,12 @@ void primitive_inst::save(cldnn::BinaryOutputBuffer& ob) const {
     const auto _allocation_type = _outputs[0]->get_allocation_type();
     ob << make_data(&_allocation_type, sizeof(_allocation_type));
 
+    bool memory_reuse_by_user = true;
+    if (user_requesting_mem_reuse_false(*_node)) {
+        memory_reuse_by_user = false;
+    }
+    ob << memory_reuse_by_user;
```

Comment:
not very important but the variable name sounds weird. The original intention was " user does not allows memory reuse" not "reused by user". So.. how about just set it "can_reuse_memory" ?

Reply:
I renamed it as `can_reuse_memory`, Thank you.